Core/Src/main.c

```diff
@@ -212,9 +249,9 @@ static void MX_TIM16_Init(void)
 
   /* USER CODE END TIM16_Init 1 */
   htim16.Instance = TIM16;
-  htim16.Init.Prescaler = 79;
+  htim16.Init.Prescaler = 19;
   htim16.Init.CounterMode = TIM_COUNTERMODE_UP;
-  htim16.Init.Period = 65535;
+  htim16.Init.Period = 41999;
```

Comment:
How many `ms` does the timer get triggered?

Reply:
Should be 10 ms

Reply:
Can you include how it is calculated as a comment?